controller/src/beerocks/master/db/db.cpp

```diff
@@ -2202,6 +2202,38 @@ bool db::get_hostap_is_acs_enabled(std::string mac)
     return n->hostap->is_acs_enabled;
 }
 
+bool db::set_hostap_passive_mode_enabled(std::string mac, bool enable)
```

Comment:
The whole of this commit should be removed. The only mode we support from now on is passive mode, so the controller doen't need to be notified.

Reply:
For now, it is needed because some parts of the code uses it.
I will remove it nicely in a different PR right after I'll finish with the merging the Turris Omnia changes.

Reply:
OK.